src/SFA.DAS.Roatp.Functions/ApplyTypes/Event.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace SFA.DAS.Roatp.Functions.ApplyTypes
+{
+    public class Event
```

Comment:
Event is generic, hope  its nice we have specific naming convention for this instance

Reply:
I'm not sure what you mean here Suresh? It's a JSON structure from an external API

Reply:
This is just creating a structure to take in the json

Reply:
I'll move this and other related classessm into a non-ApplyTypes namespace to avoid confusion